apps/electron/electron.config.js

```diff
@@ -4,12 +4,12 @@ module.exports = {
   },
   reactAppUrl: {
     development: 'http://localhost:3000/editor',
-    production: 'http://localhost:4001/editor',
+    production: (port = 4001) => `http://localhost:${port}/editor`,
```

Comment:
rather than passing the port, what do you think about having some sort of template like,

``` javascript
// in the config file
production: 'http://localhost:%port%/editor'

// and then when we use in main.js
const clientURL = electronConfig.production.replace('%port%', port)
```

Reply:
IMO I think passing the port is better as this templating isn't standardized and can create mess later on. In another place I found something like `__IP__`.

And I think it's faster to generate a string using string laterals than using `replace` as it has to first find and then replace.

I think we should also turn the `development` value to a function too.

Reply:
sounds good on the account is spaghetti code 😅  

Speed at startup is not really a concern since the difference would have been in the microsecodns